crypto/hash/hmac_ossl.c

```diff
@@ -118,38 +184,66 @@ static srtp_err_status_t srtp_hmac_dealloc(srtp_auth_t *a)
 
 static srtp_err_status_t srtp_hmac_start(void *statev)
 {
-    HMAC_CTX *state = (HMAC_CTX *)statev;
-
-    if (HMAC_Init_ex(state, NULL, 0, NULL, NULL) == 0)
+    srtp_hmac_ossl_ctx_t *hmac = (srtp_hmac_ossl_ctx_t *)statev;
+
+#ifdef SRTP_OSSL_USE_EVP_MAC
+    if (hmac->use_dup) {
+        if (hmac->ctx_dup) {
+            EVP_MAC_CTX_free(hmac->ctx_dup);
+        }
+        hmac->ctx_dup = EVP_MAC_CTX_dup(hmac->ctx);
+        if (hmac->ctx_dup == NULL) {
+            return srtp_err_status_alloc_fail;
+        }
+    } else {
+        if (EVP_MAC_init(hmac->ctx, NULL, 0, NULL) == 0)
+            return srtp_err_status_auth_fail;
```

Comment:
```suggestion
        if (EVP_MAC_init(hmac->ctx, NULL, 0, NULL) == 0) {
            return srtp_err_status_auth_fail;
        }
```

Always use curly braces around `if` statements, here and elsewhere.